h/static/scripts/group-forms/components/AppRoot.tsx

```diff
@@ -0,0 +1,48 @@
+import { Route, Switch } from 'wouter-preact';
+import { useMemo } from 'preact/hooks';
+
+import CreateEditGroupForm from './CreateEditGroupForm';
+import EditGroupMembersForm from './EditGroupMembersForm';
+import type { ConfigObject } from '../config';
+import { Config } from '../config';
+import { routes } from '../routes';
+
+export type AppRootProps = {
+  config: ConfigObject;
+};
+
+export default function AppRoot({ config }: AppRootProps) {
+  const stylesheetLinks = useMemo(
+    () =>
+      config.styles.map((stylesheetURL, index) => (
+        <link
+          key={`${stylesheetURL}${index}`}
+          rel="stylesheet"
+          href={stylesheetURL}
+        />
+      )),
+    [config],
+  );
+
+  return (
+    <>
+      {stylesheetLinks}
+      <Config.Provider value={config}>
+        <Switch>
+          <Route path={routes.groups.new}>
+            <CreateEditGroupForm />
+          </Route>
+          <Route path={routes.groups.edit}>
+            <CreateEditGroupForm />
+          </Route>
+          <Route path={routes.groups.editMembers}>
+            <EditGroupMembersForm />
+          </Route>
```

Comment:
The `/groups/:pubid/edit/members` route is available unconditionally. This means that you can access it via URL if the group members flag is turned off. The flag only controls whether the Settings / Members links are displayed on the `/groups/:pubid/edit` route.

Reply:
What are you planning to do if the URL is manually visited while the feature flag is disabled? Will you show an error in `EditGroupMembersForm` or something like that?